modules/apps/29-fee/keeper/keeper.go

```diff
@@ -24,8 +22,8 @@ var _ types.ChannelKeeper = (*Keeper)(nil)
 
 // Keeper defines the IBC fungible transfer keeper
 type Keeper struct {
-	storeService corestore.KVStoreService
-	cdc          codec.BinaryCodec
+	appmodule.Environment
```

Comment:
Dittorino (as in other PR :sweat_smile:)

```suggestion
	appmodule.Environment

```